test/unexpected-knex.spec.js

```diff
@@ -1094,7 +1094,7 @@ describe('unexpected-knex', function() {
             dontIndent`
                 expected 'select * from "foo"' to have rows satisfying
                 rows =>
-                expect(rows, 'to equal', [])
+                              expect(rows, 'to equal', [])
```

Comment:
@papandreou :laughing: :see_no_evil: cc https://bounce.depfu.com/github.com/unexpectedjs/unexpected/pull/553

Reply:
Maybe something to do with my indenting :)

Reply:
Argh, haha. Yeah, that should probably be reindented to two spaces, unless that `dontIndent` thing does something funky. I'll take a look later :)

Reply:
Fixed in unexpected 10.40.2 / 11.0.0-4 😌

Reply:
https://github.com/unexpectedjs/unexpected-knex/commit/2b6b2bc9969d40289c5c028710ac8e8dc5f05f5f :tada: